scripts/run_pier.sh

```diff
@@ -279,6 +303,8 @@ if [ "$OPT" == "up" ]; then
   pier_up
```

Comment:
`goduck pier start` command should not remove old container or remove store now, for the reason of new command `goduck pier clean`